egret/model_library/unit_commitment/uc_model_generator.py

```diff
@@ -37,7 +38,7 @@
                              ]
                             )
 
-def generate_model( model_data, uc_formulation, relax_binaries=False, ptdf_options=None, PTDF_matrix_dict=None ):
+def generate_model( model_data, uc_formulation, relax_binaries=False, ptdf_options=None, PTDF_matrix_dict=None, slack_type=SlackType.BUS_BALANCE):
```

Comment:
Shouldn't the default slack type be `SlackType.NONE`?

Reply:
To maintain the existing behavior of Egret, this is the correct default. I can make the case for all three:
1. `SlackType.BUS_BALANCE` is the existing unit commitment model
2. `SlackType.TRANSMISSION_LIMITS` is probably most common in industry practice
3. `SlackType.NONE` is probably most common in academic models.

Reply:
Got it.